src/main/java/com/google/api/generator/gapic/composer/samplecode/SampleUtil.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.gapic.composer.samplecode;
+
+import com.google.api.generator.engine.ast.*;
+import com.google.api.generator.gapic.utils.JavaStyle;
+
+public class SampleUtil {
+  public static String composeSampleMethodName(String clientName, String methodName) {
+    if (clientName.equals("") || methodName.equals("")) {
+      throw new IllegalArgumentException("clientName and methodName must exist");
+    }
+    return JavaStyle.toLowerCamelCase(clientName + JavaStyle.toUpperCamelCase(methodName));
+  }
+
+  public static MethodInvocationExpr systemOutPrint(String content) {
+    return composeSystemOutPrint(ValueExpr.withValue(StringObjectValue.withValue(content)));
+  }
+
+  public static MethodInvocationExpr systemOutPrint(VariableExpr variableExpr) {
+    return composeSystemOutPrint(variableExpr.toBuilder().setIsDecl(false).build());
+  }
+
+  static MethodInvocationExpr composeSystemOutPrint(Expr content) {
```

Comment:
These method are not called by non-test code. Let's remove them.

Reply:
That's actually what I suspected. Let's add them in the future when they are needed.